kellydinneen/stretch
# Feature/dynamic flower sizing

## Is this a fix or a feature?
- Fix and Feature

## What is the change/fix?
- Flower petals render based on code lines, not active life
- Home logo renders on profile vis, "GitHub Garden" renders on home
- Added to the key to have most common coding languages

## Where should the reviewer start?
- Home.js
- Header.js
- Garden.js

## How should this be tested?
- Verify the conditional rendering of the logo and check the variation in flower petal sizing

## Screenshots(if appropriate)

## This PR is related to which issues:
Closes #23 
Closes #42
